statistics.md

```diff
@@ -148,13 +148,13 @@ If a table has many columns, collecting statistics on all the columns can cause
 
         <CustomContent platform="tidb">
 
-        After the setting, TiDB writes the `PREDICATE COLUMNS` information to the `mysql.column_stats_usage` system table every 100 * [`stats-lease`](/tidb-configuration-file.md#stats-lease).
+        After the setting, TiDB writes the `PREDICATE COLUMNS` information to the [`mysql.column_stats_usage`](/mysql-schema.md#statistics-system-tables) system table every 100 * [`stats-lease`](/tidb-configuration-file.md#stats-lease).
```

Comment:
The docs for the `mysql` schema could be improved:
- One page per table, just like `information_schema` etc.
- https://github.com/pingcap/docs/issues/16684

But this doesn't need to be fixed in this PR